libcxx/test/std/time/time.zone/time.zone.timezone/time.zone.members/get_info.local_time.pass.cpp

```diff
@@ -13,6 +13,9 @@
 // XFAIL: libcpp-has-no-experimental-tzdb
 // XFAIL: availability-tzdb-missing
 
+// TODO TZDB Investigate why this fails.
+// UNSUPPORTED: target={{.*}}
```

Comment:
It would be nice to have more information about exactly _what_ fails and how to reproduce it.